### ISSUE TITLE
autoconf:not found while compiling corebird

### ISSUE THREAD

**Issue by moein-rahimi** [baedert/corebird#503]
Hi
I have issue with compiling the corebird, when i try this code : 
`./autogen.sh --prefix=/usr`

i got this message:
```bash
./autogen.sh: 1: ./autogen.sh: autoconf: not found
+ aclocal --install
./autogen.sh: 22: ./autogen.sh: aclocal: not found
+ exit 1
```

**Comment by baedert** [comment]
Install whatever package provides the "autotools" executable in your distribution.

**Comment by moein-rahimi** [comment]
i'm using ubuntu 15.10, i just installed automake, autoconf and intltool
now i got this message
````bash
+ aclocal --install
aclocal: installing 'm4/intltool.m4' from '/usr/share/aclocal/intltool.m4'
aclocal: installing 'm4/nls.m4' from '/usr/share/aclocal/nls.m4'
+ intltoolize --force --copy --automake
+ autoreconf --verbose --force --install -Wno-portability
autoreconf: Entering directory `.'
autoreconf: configure.ac: not using Gettext
autoreconf: running: aclocal --force --warnings=no-portability -I m4 ${ACLOCAL_FLAGS}
autoreconf: configure.ac: tracing
autoreconf: configure.ac: creating directory build-aux
autoreconf: configure.ac: not using Libtool
autoreconf: running: /usr/bin/autoconf --force --warnings=no-portability
autoreconf: running: /usr/bin/autoheader --force --warnings=no-portability
autoreconf: running: automake --add-missing --copy --force-missing --warnings=no-portability
configure.ac:16: installing 'build-aux/compile'
configure.ac:10: installing 'build-aux/install-sh'
configure.ac:10: installing 'build-aux/missing'
src/Makefile.am:27: error: Libtool library used but 'LIBTOOL' is undefined
src/Makefile.am:27:   The usual way to define 'LIBTOOL' is to add 'LT_INIT'
src/Makefile.am:27:   to 'configure.ac' and run 'aclocal' and 'autoconf' again.
src/Makefile.am:27:   If 'LT_INIT' is in 'configure.ac', make sure
src/Makefile.am:27:   its definition is in aclocal's search path.
src/Makefile.am: installing 'build-aux/depcomp'
parallel-tests: installing 'build-aux/test-driver'
autoreconf: automake failed with exit status: 1
+ exit 1
````

**Comment by baedert** [comment]
Now install libtool... that's really the answer everytime you get an error like that, you just need to install the package.

**Comment by moein-rahimi** [comment]
ok now i got this message 
````bash
+ aclocal --install
aclocal: installing 'm4/gsettings.m4' from '/usr/share/aclocal/gsettings.m4'
+ intltoolize --force --copy --automake
+ autoreconf --verbose --force --install -Wno-portability
autoreconf: Entering directory `.'
autoreconf: configure.ac: not using Gettext
autoreconf: running: aclocal --force --warnings=no-portability -I m4 ${ACLOCAL_FLAGS}
autoreconf: configure.ac: tracing
autoreconf: running: libtoolize --copy --force
libtoolize: putting auxiliary files in AC_CONFIG_AUX_DIR, `build-aux'.
libtoolize: copying file `build-aux/ltmain.sh'
libtoolize: putting macros in AC_CONFIG_MACRO_DIR, `m4'.
libtoolize: copying file `m4/libtool.m4'
libtoolize: copying file `m4/ltoptions.m4'
libtoolize: copying file `m4/ltsugar.m4'
libtoolize: copying file `m4/ltversion.m4'
libtoolize: copying file `m4/lt~obsolete.m4'
autoreconf: running: /usr/bin/autoconf --force --warnings=no-portability
autoreconf: running: /usr/bin/autoheader --force --warnings=no-portability
autoreconf: running: automake --add-missing --copy --force-missing --warnings=no-portability
autoreconf: Leaving directory `.'
+ set +x
+ ./configure --prefix=/usr
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /bin/mkdir -p
checking for gawk... no
checking for mawk... mawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking whether make supports nested variables... (cached) yes
checking whether to enable maintainer-specific portions of Makefiles... yes
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking whether gcc understands -c and -o together... yes
checking for style of include used by make... GNU
checking dependency style of gcc... gcc3
checking for gcc option to accept ISO C99... none needed
checking for gcc option to accept ISO Standard C... (cached) none needed
checking build system type... x86_64-unknown-linux-gnu
checking host system type... x86_64-unknown-linux-gnu
checking how to print strings... printf
checking for a sed that does not truncate output... /bin/sed
checking for grep that handles long lines and -e... /bin/grep
checking for egrep... /bin/grep -E
checking for fgrep... /bin/grep -F
checking for ld used by gcc... /usr/bin/ld
checking if the linker (/usr/bin/ld) is GNU ld... yes
checking for BSD- or MS-compatible name lister (nm)... /usr/bin/nm -B
checking the name lister (/usr/bin/nm -B) interface... BSD nm
checking whether ln -s works... yes
checking the maximum length of command line arguments... 1572864
checking whether the shell understands some XSI constructs... yes
checking whether the shell understands "+="... yes
checking how to convert x86_64-unknown-linux-gnu file names to x86_64-unknown-linux-gnu format... func_convert_file_noop
checking how to convert x86_64-unknown-linux-gnu file names to toolchain format... func_convert_file_noop
checking for /usr/bin/ld option to reload object files... -r
checking for objdump... objdump
checking how to recognize dependent libraries... pass_all
checking for dlltool... no
checking how to associate runtime and link libraries... printf %s\n
checking for ar... ar
checking for archiver @FILE support... @
checking for strip... strip
checking for ranlib... ranlib
checking command to parse /usr/bin/nm -B output from gcc object... ok
checking for sysroot... no
checking for mt... mt
checking if mt is a manifest tool... no
checking how to run the C preprocessor... gcc -E
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking for dlfcn.h... yes
checking for objdir... .libs
checking if gcc supports -fno-rtti -fno-exceptions... no
checking for gcc option to produce PIC... -fPIC -DPIC
checking if gcc PIC flag -fPIC -DPIC works... yes
checking if gcc static flag -static works... yes
checking if gcc supports -c -o file.o... yes
checking if gcc supports -c -o file.o... (cached) yes
checking whether the gcc linker (/usr/bin/ld -m elf_x86_64) supports shared libraries... yes
checking whether -lc should be explicitly linked in... no
checking dynamic linker characteristics... GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... no
checking for valac... /usr/bin/valac
checking whether /usr/bin/valac is at least version 0.28.0... yes
checking for pkg-config... /usr/bin/pkg-config
checking pkg-config is at least version 0.16... yes
checking for CB... no
configure: error: Package requirements (gtk+-3.0 >= 3.16
             glib-2.0 >= 2.44
             rest-0.7 >= 0.7.93
             libsoup-2.4
             gee-0.8
             json-glib-1.0
             sqlite3 gstreamer-video-1.0 >= 1.6) were not met:

No package 'gtk+-3.0' found
No package 'rest-0.7' found
No package 'libsoup-2.4' found
No package 'gee-0.8' found
No package 'json-glib-1.0' found
No package 'sqlite3' found
No package 'gstreamer-video-1.0' found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Alternatively, you may set the environment variables CB_CFLAGS
and CB_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.
+ exit 1
````
shoud i install these above packages ? gtk json sqlite and...?

**Comment by baedert** [comment]
Yes.

**Comment by moein-rahimi** [comment]
there is a problem i got these packages installed already

**Comment by baedert** [comment]
You need the -dev or -devel versions or whatever your distro calls them.

**Comment by baedert** [comment]
... did it work out?

**Comment by moein-rahimi** [comment]
yes finally i did it !
thank you for helping.

**Comment by andersbateva** [comment]
Installing developer versions solved my problems on Ubuntu 16.04: `rest-0.7` is `librest-dev`; `sqlite3` is `libsqlite3-dev`; and `json-glib-1.0` is `libjson-glib-dev`.
